src/main/app/case/definition.ts

```diff
@@ -2518,6 +2545,19 @@ export const enum NoResponseSearchOrDispense {
   DISPENSE = 'dispense',
 }
 
+export const enum AlternativeOptions {
```

Comment:
I'd use some naming here that explains what the options are for:
```suggestion
export const enum SendPapersMethod
```